it/controllers/ReturnSentControllerIntegrationSpec.scala

```diff
@@ -38,7 +38,7 @@ class ReturnSentControllerIntegrationSpec extends Specifications with TestConfig
 //      val userAnswers = smallProducerDetaisPartialAnswers.success.value
 //      setUpData(userAnswers.copy(submitted = false))
 //      given
-//        .commonPrecondition
+//        .commonPrecondition(aSubscription)
 //
 //      WsTestClient.withClient { client =>
 //        val result1 = client.url(s"$baseUrl/return-sent")
```

Comment:
Can these be removed?

Reply:
this is christines pr. she will need it i guess